analytics/src/config/constants.js

```diff
@@ -1,79 +1,67 @@
-  ACCEPT_USER_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/accept",
-  GET_USERS_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/",
+    VERIFY_TOKEN_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/reset",
+    UPDATE_PWD_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/updatePasswordViaEmail",
+    FORGOT_PWD_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/forgotPassword",
+    LOGIN_USER_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/loginUser",
+    REGISTER_USER_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/registerUser",
+    REGISTER_CANDIDATE_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/registerCandidate",
+    REJECT_USER_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/deny",
+    ACCEPT_USER_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/accept",
+    GET_USERS_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/",
 };
 const prodConfig = {
-  VERIFY_TOKEN_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/reset",
-  UPDATE_PWD_URI:
-    "https://airqo-250220.uc.r.appspot.com/api/v1/users/updatePasswordViaEmail",
-  FORGOT_PWD_URI:
-    "https://airqo-250220.uc.r.appspot.com/api/v1/users/forgotPassword",
-  LOGIN_USER_URI:
-    "https://airqo-250220.uc.r.appspot.com/api/v1/users/loginUser",
-  REGISTER_USER_URI:
-    "https://airqo-250220.uc.r.appspot.com/api/v1/users/registerUser",
-  REGISTER_CANDIDATE_URI:
-    "https://airqo-250220.uc.r.appspot.com/api/v1/users/registerCandidate",
-  REJECT_USER_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/deny",
-  ACCEPT_USER_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/accept",
-  GET_USERS_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/",
+    VERIFY_TOKEN_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/reset",
+    UPDATE_PWD_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/updatePasswordViaEmail",
+    FORGOT_PWD_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/forgotPassword",
+    LOGIN_USER_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/loginUser",
+    REGISTER_USER_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/registerUser",
+    REGISTER_CANDIDATE_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/registerCandidate",
+    REJECT_USER_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/deny",
+    ACCEPT_USER_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/accept",
+    GET_USERS_URI: "https://airqo-250220.uc.r.appspot.com/api/v1/users/",
 };
 const defaultConfig = {
-  PORT: process.env.PORT || 5000,
-  NODE_PATH: process.env.NODE_PATH || "src",
+    PORT: process.env.PORT || 5000,
+    NODE_PATH: process.env.NODE_PATH || "src",
 };
 
 function envConfig(env) {
-  switch (env) {
-    case "development":
-      return devConfig;
-    case "test":
-      return testConfig;
-      case "stage":
-      return stageConfig;
-    default:
-      return prodConfig;
-  }
+    switch (env) {
```

Comment:
A temp fix to the unhandled rejection error. For some reason, error handling implementations was being skipped. So for now, please update (git pull) this branch on your local machine and proceed with the tests like a real boss. At this point, you will connect directly to the DB instance in the cloud. @sserurich